plugins/amd-kria/fu-amd-kria-plugin.c

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright 2024 Advanced Micro Devices Inc.
+ * All rights reserved.
+ *
+ * This file is provided under a dual MIT/LGPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ * AMD Chooses the MIT license part of Dual MIT/LGPLv2 license agreement.
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR MIT
+ */
+
+#include "config.h"
+
+#include "fu-amd-kria-plugin.h"
+#include "fu-amd-kria-device.h"
+#include "fu-amd-kria-image-reg.h"
+#include "fu-amd-kria-persistent-reg.h"
+#include "fu-amd-kria-som-eeprom.h"
+
+struct _FuAmdKriaPlugin {
+	FuPlugin parent_instance;
+	gchar *version_a;
+	gchar *version_b;
+	gboolean image_a;
```

Comment:
I think you should print this at startup with the to_string vfunc.

Reply:
added